src/chain_spec.rs

```diff
@@ -288,7 +294,7 @@ fn genesis_processed_data(
         .map(|(BridgeLockId { nonce, tx_hash }, recipient)| BridgeTx {
             nonce,
             recipient,
-            amount: BOOTSTRAP_STASH,
+            amount: BOOTSTRAP_TREASURY,
```

Comment:
I think it should be bootstrap_stash as the other variable is used the bootstrapping the treasury balance

Reply:
as mentioned in the PR description, the higher balance is needed for the integration tests, another approach would be to setBalance in the init function of the integration tests, I tried this but I couldn't get it to work (`bad origin` even though I was using Alice which in theory is root) I thought this change would be fine since it only affects testnets but if it's not, I'll figure out how to make it work a different way

Reply:
If we need more balance then we can just bump up the value under the same name variable 😉 . Instead of switching the variable itself.  Although I am good to merge as it is as well

Reply:
Not really because that variable is used in other places, so I'd either leave it like this or create a different variable, I think creating a different variable is probably better so I'll do that